packages/starlight/utils/createTranslationSystem.ts

```diff
@@ -56,7 +56,7 @@ export function createTranslationSystem<T extends i18nSchemaOutput>(
 	 * - `exists()` method for checking if a key exists in the dictionary.
 	 * - `dir()` method for getting the text direction of the locale.
 	 *
-	 * @param {string | undefined} [locale]
+	 * @param {string | undefined} [lang]
```

Comment:
The comment description on line 52 also needs updating (thanks GitHub for not letting me suggest the change directly).